Source/SwiftLintFramework/Rules/StatementPositionRule.swift

```diff
@@ -28,15 +28,59 @@ public struct StatementPositionRule: Rule {
             "}↓  else {",
             "}↓\ncatch {",
             "}\n\t↓  catch {"
+        ],
+        corrections: [
+          "}\n else {\n":"} else {\n",
```

Comment:
we also use 4-space indentation. please update all your changes in this file to match that